docs/computing/jobs/partitions.md

```diff
@@ -1,40 +1,50 @@
 # Available Partitions
 
-[lumid]: ..//systems/lumid.md
+[lumi-c]: ../../computing/systems/lumic.md
+[lumi-g]: ../../computing/systems/lumig.md
+[lumi-d]: ../../computing/systems/lumid.md
 
-The partition setup of LUMI prioritizes jobs that aim to scale out. As a
+The Slurm partition setup of LUMI prioritizes jobs that aim to scale out. As a
 consequence most nodes are reserved for jobs that use all available resources.
 However, some nodes are reserved for smaller allocations and debugging. 
 
-## Partitions allocatable by node
+!!! info
```

Comment:
I would consider this text a bit, but don't have a better suggestion just now.

Reply:
I have removed this info box as I realized that it might not be entirely accurate depending on the final partition layout when LUMI-G becomes GA.